cves/CVE-2011-0722.yml

```diff
@@ -270,8 +310,10 @@ i18n:
 
     Answer should be boolean. Write a note about how you came to the conclusions
     you did.
-  answer:
-  note:
+  answer: false
+  note: |
+    The vulnerability was in an audio-video decoder, which is agnostic to
```

Comment:
"agnostic" is a bit of an odd phrasing here, reword?

Reply:
Will do.

Reply:
Update added